# program to generate Arithmetic Exception without exception handling
num1=10
num2=0
res=num1/num2  
print("Result:", res)
